td.vue/src/components/GraphButtons.vue

```diff
@@ -87,20 +87,30 @@ export default {
             return;
         },
         undo() {
-            if (this.graph.canUndo()) {
-                this.graph.undo();
+            if (this.graph.getPlugin('history').canUndo()) {
+                this.graph.getPlugin('history').undo();
```

Comment:
are we sure that we need the getPlugin() @lreading ? It works with or without using it

Reply:
It is not working on my machine without the `getPlugin`.  The history property is always undefined. :thinking: 
I wonder if there's somehow something different about our environments?

Reply:
good point, I am on MacOS/ARM64 so it may well be different
it works for me with and without call to getPlugin so I agree we leave it as you have it now